xml/admin_saltcluster.xml

```diff
@@ -1376,6 +1376,226 @@ OSD_ID  HOST         STATE                    PG_COUNT  REPLACE  FORCE  STARTED_
    </para>
   </note>
 
+  <sect2 xml:id="updating-ceph-https-proxy">
+   <title>Configuring HTTPS proxy</title>
+   <para>
+    &ceph; clusters are supposed to use an HTTPS proxy to communicate with the
+    container registry server. To configure the proxy, follow these steps on
+    the &adm;:
+   </para>
+   <procedure>
+    <step>
+     <para>
+      Copy the configuration file for containers:
+     </para>
+<screen>&prompt.sudo;cp /usr/share/containers/containers.conf /etc/containers/containers.conf</screen>
+    </step>
+    <step>
+     <para>
+      Edit the newly copied file and add the <option>http_proxy</option>
+      setting to its <literal>[engine]</literal> section, for example:
+     </para>
+<screen>&prompt.user;cat /etc/containers/containers.conf
+[engine]
+http_proxy=proxy.example.com
+[...]
+</screen>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="updating-ceph-local-registry">
+   <title>Creating the local registry (optional)</title>
+   <para>
+    Although program temporary fixes (PTFs) are published to the
+    <literal>registry.suse.com</literal> server, creating local container
+    registry may be useful for several scenarios, for example, air-gapped
+    deployments.
+   </para>
+   <tip>
+    <title>Placement and used port</title>
+    <para>
+     Deploy the registry on a machine accessible by all nodes in the cluster.
+     We recommend the &adm;. By default, the registry listens on port 5000. On
+     the registry node, use the following command to ensure that the port is
+     free:
+    </para>
+<screen>ss -tulpn | grep :5000</screen>
+    <para>
+     If other processes (such as <literal>iscsi-tcmu</literal>) are already
+     listening on port 5000, determine another free port which can be used to
+     map to port 5000 in the registry container.
+    </para>
+   </tip>
+   <procedure>
+    <step>
+     <para>
+      Verify that the <literal>Containers Module</literal> extension is
```

Comment:
Suggestion: is the [package] tag more appropriate here?

Reply:
no, it's not a package, it's an add-on. maybe <systemitem>?

Reply:
....but maybe yes :-)